packages/mobile/src/transactions/TransferConfirmationCard.tsx

```diff
@@ -178,9 +179,14 @@ function PaymentReceivedContent({ address, recipient, e164PhoneNumber, amount, c
   const totalAmount = amount
   const isCeloTx = amount.currencyCode === CURRENCIES[CURRENCY_ENUM.GOLD].code
   const celoEducationUri = useTypedSelector((state) => state.app.celoEducationUri)
+  const rewardsEnabled = useTypedSelector((state) => state.account.rewardsEnabled)
 
   const openLearnMore = () => {
-    navigate(Screens.WebViewScreen, { uri: celoEducationUri! })
```

Comment:
This link was actually fine since this redirects to a CELO education webpage on CELO transfers.

The one we need to change is in line 215 in this same file:
```
  const openLearnMore = () => {
    navigate(Screens.ConsumerIncentivesHomeScreen)
    ValoraAnalytics.track(RewardsEvents.rewards_screen_opened, {
      origin: RewardsScreenOrigin.PaymentDetail,
    })
  }
``` 

That's what's used for CELO rewards transfers, so in there we want:
- If the user has `rewardsEnabled`, then don't change anything, open the `ConsumerIncentivesHomeScreen` screen.
- If the user doesn't have `rewardsEnabled`, then  it's a user that had already been participating in the rewards program but no longer qualifies, so we need to open in a WebView the link `CELO_REWARDS_LINK` (it's already in the config file) which is the terms and conditions of the old rewards program.